server/schedule/operator_controller.go

```diff
@@ -210,7 +219,7 @@ func (oc *OperatorController) AddOperator(ops ...*Operator) bool {
 
 	if oc.exceedStoreLimit(ops...) || !oc.checkAddOperator(ops...) {
 		for _, op := range ops {
-			operatorCounter.WithLabelValues(op.Desc(), "canceled").Inc()
+			operatorCounter.WithLabelValues(op.Desc(), "cancel").Inc()
```

Comment:
We also need to make a change in `tidb-ansible` for both `master` and `release-3.0` branches